t/consistent-hashed-upstream.t

```diff
@@ -0,0 +1,90 @@
+# vi:filetype=
+
+use lib 'lib';
+use Test::Nginx::Socket;
+
+#repeat_each(3);
```

Comment:
Basically the layout of test case is following [hashed-upstream.t]((https://github.com/openresty/set-misc-nginx-module/blob/master/t/hashed-upstream.t)